### DISCUSSION TITLE
Interview attempts failing

### DISCUSSION THREAD

**Discussion by Track0** [zwave-js/zwave-js-ui#337] [Q&A]
I have been trying to get this working for a couple days. I have an Aeotec USB zwave stick. I have zwavejs2mqtt running in a docker container. I am trying to connect Inovelli red series on/off switch. I have tried multiple different switches. I am able to connect the node, it starts gathering info, and collects some of it, but always dies after the line "INFO ZWAVE: Node 3: metadata updated: 117-0-rf" in the logs with an error like this "ERROR ZWAVE: Interview attemp 4 of node 3 has failed: Attempt 4/5 failed", obviously the error changes depending on the node or the retry number. It always dies in the same spot on the logs.

I have tried rebuilding the docker container multiple times, with no luck. Any thoughts or suggestions would be great.

![image](https://user-images.githubusercontent.com/6044670/105659039-6e5e3700-5e8d-11eb-851c-2dac79100210.png)
![image](https://user-images.githubusercontent.com/6044670/105659074-80d87080-5e8d-11eb-952f-a2c97625a66c.png)
![image](https://user-images.githubusercontent.com/6044670/105659089-8df55f80-5e8d-11eb-9e49-59460dcfb332.png)

These are the settings I am currently using.
Thanks

**Comment by robertsLando** [comment]
@Track0 I need the zwavejs logs file. Enable logs in zwave section and also enable the log to file. Then go to the store explorer and export it

**Reply by Track0** [reply]
@robertsLando this is the log that I ran for a few hours.
[zwavejs_1.log](https://github.com/zwave-js/zwavejs2mqtt/files/5867904/zwavejs_1.log)

**Reply by robertsLando** [reply]
What's the logLevel you used? Silly verbose ? @AlCalzone COuld you give a look at the logs?

**Reply by Track0** [reply]
There wasn't a logging level listed under zwave, but the logging level under general is info. I can go verbose or silly if that might help.

**Reply by AlCalzone** [reply]
To debug that, I need `debug` or `silly`

**Reply by Track0** [reply]
@AlCalzone I switched to silly, and let it run until the third attempt.

[zwavejs_1.log](https://github.com/zwave-js/zwavejs2mqtt/files/5868079/zwavejs_1.log)

**Reply by AlCalzone** [reply]
That does not look like `silly`

**Reply by Track0** [reply]
Sorry about that, I made the change in the wrong place. I am generating a new log now. I will upload it shortly.

**Reply by Track0** [reply]
@AlCalzone this log looks a little more silly
[zwavejs_1.log](https://github.com/zwave-js/zwavejs2mqtt/files/5868309/zwavejs_1.log)

**Reply by Track0** [reply]
@AlCalzone Just for a little more information, I have tried swapping to another USB Zwave stick, and the issue persists.

**Reply by Track0** [reply]
@AlCalzone or @robertsLando any thoughts on this? I need to get this working, and really like the idea of this being open source and in node. I would love to learn enough about it to troubleshoot this deeper, but I don't have the time right now.

**Reply by AlCalzone** [reply]
Sorry - forgot to ping here too. Try https://github.com/zwave-js/node-zwave-js/pull/1505 please.
Instructions for docker @ https://zwave-js.github.io/zwavejs2mqtt/#/troubleshooting/docker_custom_builds

**Reply by Track0** [reply]
I'll give that a shot this evening when I have some time, and let you know here.
